modules/nf-core/sgdemux/meta.yml

```diff
@@ -0,0 +1,64 @@
+name: "sgdemux"
+description: Demultiplex bgzip'd fastq files
+keywords:
+  - demultiplex
+  - fastq
+tools:
+  - "sgdemux":
+      description: "Tool for demultiplexing sequencing data generated on Singular Genomics' sequencing instruments."
+      homepage: "https://github.com/Singular-Genomics/singular-demux"
+      documentation: "https://github.com/Singular-Genomics/singular-demux"
+      licence: "['For Singular G4™ Sequencing Platform only']"
+
+input:
+- meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - run_manifest:
+      type: file
+      description: RunManifest file
+      pattern: "*.{csv}"
+  - run_dir:
+      type: directory
+      description: "Input run directory containing optionally containing a RunManifest.json if run_manifest is not supplied"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - sample_fastq:
+      type: file
+      description: Demultiplexed sample FASTQ files
+      pattern: "output/*_R*.fastq.gz"
+  - metrics:
```

Comment:
update here and below based on https://github.com/Singular-Genomics/singular-demux#metrics